core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationService.java

```diff
@@ -390,7 +390,7 @@ public FlowIdentifier updateNodeCount(StackDto stackDto, InstanceGroupAdjustment
     FlowIdentifier start(StackView stack) {
```

Comment:
the jira ticket talks about `stop/start/create/delete` operations but here I can only see `start`, are other operations supported?

Reply:
Stop and delete already works before this change. Added it in the description